### PR TITLE
edit new content

### PR DESCRIPTION
**Pull request by cawrites**
updated contributor guide, added images. created small file for Hactoberfest

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -20,16 +20,72 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
...
-  - Give sample code if you can
-- What you expected would happen
-- What actually happens
-- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+ [![Select button for issue](images/new-issue.png)](https://github.com/blaccsmith/website/issues)
 
-People _love_ thorough bug reports. I'm not even kidding.
+ Select **Get Started** for Bug Report.
+
+  [![Select button for bug report](images/bug-and-feature-request.png)](https://github.com/blaccsmith/website/issues)
```
Comment (blaccsmith) [inline .github/CONTRIBUTING.md:31]: Hey nice updates, however this button is trying to render as a picture. Could you edit this

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -20,16 +20,72 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
...
+
+**Additional context**
+Add any other context about the problem here.
+```
+
+## Request a new feature
+
+To request a new feature follow the same path as a bug report and select **Getting Started** for feature request.
+
+[![Select button for new feature](images/bug-and-feature-request.png)](https://github.com/blaccsmith/website/issues)
```
Comment (blaccsmith) [inline .github/CONTRIBUTING.md:66]: Same as above comment

**Review by blaccsmith — DISMISSED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (cawrites) [reply .github/CONTRIBUTING.md:31]: I embedded a link to the image. I can remove it.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (cawrites) [reply .github/CONTRIBUTING.md:66]: Hmm that's just supposed to be a description for the image. I will remove it.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (cawrites) [reply .github/CONTRIBUTING.md]: this has been resolved

**Comment by cawrites** [comment]
@blaccsmith can you merge this.

File: .github/Hacktoberfest.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# 🙌 Hacktoberfest
```
Comment (blaccsmith) [inline .github/Hacktoberfest.md:1]: Per the previous team meeting can we move this into the top of the Contributing.md for the time being !

**Review by blaccsmith — CHANGES_REQUESTED** [review]

File: .github/Hacktoberfest.md (hunk 3 above)
Comment (cawrites) [reply .github/Hacktoberfest.md:1]: change completed

**Review by blaccsmith — APPROVED** [review]
Looks good ! Thank you @cawrites

**Pull request merged**